obspy/io/rg16/core.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import numpy as np
+from obspy.core import Stream, Trace, Stats, UTCDateTime
+
+from .util import _read, open_file, _read_block, _quick_merge
+
+# --------------------- define specs of needed blocks
+
+
+# blocks are specified as a list of tuples. Each tuple contains the following:
+# (name, startbyte, length, format), explanation as follows:
+# name - The name, these will be keys in a returned dict
+# startbye - the byte position, relative to the block, to start reading
+# length - the number of bytes to read
+# format - format to interpret the data being read. see rg16.utils.read.
+
+
+# header block, combines header block one and two
+general_header_block = [
+    ('channel_sets', 28, 1, 'bcd'),
+    ('num_additional_headers', 11, 1, '>i.'),
+    ('extended_headers', [30, 37], [1, 2], ['bcd', '>i2']),
+    ('external_headers', [31, 39], [1, 3], ['bcd', '>i3']),
+    ('record_length', 46, 3, '>i3'),
+    ('base_scan', 22, 1, '>i1'),  # https://imgur.com/a/4aneG
```

Comment:
The two images not included show how base scan intervals are mapped to sampling rate, and the appropriate datatype for the time series. Both of these are handled by the code so it is not very important the users have access to them. Maybe I should remove  the link since the important figure is now included?

Reply:
You could replace the link to point to the image in the docs..

Also, if you feel the information in the other two pics might be important, I think it could easily be included in a ASCII comment right here